src/test/scala/org/kframework/minikore/TextToMiniTest.scala

```diff
@@ -281,7 +281,7 @@ class TextToMiniTest {
       case ParseError(msg) =>
         assertEquals(
           strip("""
-            |ERROR: Line 3: Column 11: Expected \true, \false, \and, \or, \not, \implies, \exists, \forall, \next, \rewrite, or \equal, but 't '
+            |ERROR: Line 3: Column 11: Expected \true, \false, \and, \or, \not, \implies, \exists, \forall, \next, \rewrite, or \equal, but '\t '
```

Comment:
I actually found it confusing the other way, without a `\`.  The next line shows the exact position anyway.  I am OK either way.